src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -46,6 +46,15 @@ public class DeleteCommand extends UndoableCommand {
     public static final String MESSAGE_PERSON_NOT_FOUND =
             "The given combination of Name and NRIC does not match any patient in the patients list.";
 
+    public static final String MESSAGE_NO_APPOINTMENT_TO_DELETE = "Patient does not have an appointment to delete";
+
+    public static final String MESSAGE_NO_MEDICAL_HISTORY_TO_DELETE =
+            "Patient does not have any medical histories to delete";
+
+    public static final String MESSAGE_INVALID_MEDICAL_HISTORY =
+            "Patient does not have the medical histories specified.\n"
+            + "Please check the medical histories of the patient through finding the patient first.";
```

Comment:
The sentence `Please check the medical histories of the patient through finding the patient first` is a bit confusing. I think it would be better to remove the entire sentence.